python/cuml/cuml/testing/utils.py

```diff
@@ -364,44 +364,22 @@ def get_models(self):
         return models
 
 
-def get_classes_from_package(package, import_sub_packages=False):
+def get_all_base_subclasses() -> dict[str, type]:
```

Comment:
The old function was only used to find all subclasses of `Base`. There's a much easier way to do that, I've done it here. I can confirm that the results are identical to before, except that here I took the opportunity to filter out subclasses with a leading `_` since those didn't make sense to test. Otherwise this is just a drop-in to the call sites from before but without the directory traversing (which led to accidental import of `dask`).